### DISCUSSION TITLE
Rendering problem: DatePicker -> showIcon=true -> ui-fluid

### DISCUSSION THREAD

**Discussion by mileslaw** [primefaces/community#1689] [PrimeFaces Templates] [answered]
Hello,
datePicker icon isn't showing properly when used in combination with primeflex (form layout and ui-fluid) .

example below:
```
  <div class="formgrid grid ui-fluid">
                   <div class="field col-6" >
                        <p:outputLabel value="Data urodzenia od" for="dataUrodzeniaOd"/>
                        <p:datePicker id="dataUrodzeniaOd" value="#{parametry.dataUrodzeniaOd}" placeholder="Data urodzenia od"    showIcon="true"/>
                    </div>
  </div>
```
Renders like this:
![Zrzut ekranu z 2024-04-17 10-57-28](https://github.com/primefaces/community/assets/6030847/15426a4a-90d7-4037-9a50-d8c3fe702023)

**Comment by melloware** [comment, marked as answer]
Here is a PF reproducer:
[discussion-1689.zip](https://github.com/primefaces/community/files/15047978/discussion-1689.zip)

Run `mvn clean jetty:run` and go to http://localhost:8080/ and you will see it looks fine for me:

![image](https://github.com/primefaces/community/assets/4399574/ea741788-26fe-4cfb-a54e-ccc0732d4aa3)

So what version of PF are you using?  Sounds like your theme.css needs updating.
